apps/protocol-frontend/src/components/VerificationHover.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Tooltip } from '@chakra-ui/react';
+import React from 'react';
+
+interface VerificationHoverProps {
+  status: 'Verified' | 'Unverified' | 'noFramework';
+  threshold?: number | null;
+  children: React.ReactNode;
+}
+
+const VerificationHover = ({
+  status,
+  threshold,
+  children,
+}: VerificationHoverProps) => {
+  const verificationPropsMap = {
+    Verified: {
+      label:
+        'Feel free to attest to this contribution! Even though it has been verified for this DAO, additional attestations is good. ',
+      variant: 'tertiary',
+    },
+    Unverified: {
+      label:
+        threshold === 1
+          ? `This contribution only needs ${threshold} more attestation to be verified by your DAO! Be the one to push it through!`
```

Comment:
I have this saying `null more attestation`

Reply:
Also attestation may need to be plural in some situations

Reply:
Also, all contributions have pill even if the dao does not have a verification framework

Reply:
I think that was part of the design -- at least as I understood it

Reply:
Good catch on the `null` and the plurals -- not sure why I didn't see that locally since I tested with null and multiple attestations. Will need to look at that

Reply:
<img width="252" alt="image" src="https://user-images.githubusercontent.com/9438776/228592689-05f24edf-2432-411f-ad2b-886a622c7cb8.png">

I think all the DAOs are pills. @twblack88 should this be?

Reply:
If all the statuses are pills that is probably fine, but they have the message saying you are `x attestations away` so maybe we have a different message?

Reply:
I think I'm still unclear. My understanding is the status pill shows that message, and then the DAO column shows the checkmark on the DAO name only when it's verified.

The status pill should be showing either way -- is that not what's happening?

Reply:
![attestation-table-vf-test](https://user-images.githubusercontent.com/9438776/228595599-313856e7-feea-4613-90c7-543d280e2f4b.gif)

Reply:
Ok thanks, that helps narrow that down so I can replicate and handle the case better. I thought I tested  for both of those cases